inc/Logger.hpp

```diff
@@ -33,11 +33,12 @@ class Logger {
 private:
 	// private constructor to prevent direct object creation
 	Logger();
+	// private destructor to prevent direct object deletion
+	~Logger();
 
 	// disallow copying the instance
 	Logger(const Logger& ref);
```

Comment:
The declarations of copy ctor and assign operator can also be removed. Still compiles.

Reply:
Same thinking as with destructor: Logger is a singleton, nobody should be able to make a copy of it.